DESCRIPTION

```diff
@@ -53,6 +53,7 @@ Imports:
     rlang (>= 0.4.4),
     tidyselect (>= 1.0.0)
 Suggests:
+    pharmaversesdtm,
     admiral.test,
```

Comment:
do we still need this anymore now we've replaced with pharmaversesdtm?

Reply:
Before the release we should remove it. But `admiral.test` is still used in the ADTR vignette. It is updated in https://github.com/pharmaverse/admiralonco/issues/247.